instrumentation/netty/netty-4.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/netty/v4_0/NettyChannelPipelineInstrumentation.java

```diff
@@ -103,15 +96,4 @@ public static void addHandler(
       }
     }
   }
-
-  @SuppressWarnings("unused")
-  public static class ChannelPipelineConnectAdvice {
-
-    @Advice.OnMethodEnter
-    public static void addParentSpan(@Advice.This ChannelPipeline pipeline) {
-      Context context = Java8BytecodeBridge.currentContext();
-      Attribute<Context> attribute = pipeline.channel().attr(AttributeKeys.CONNECT_CONTEXT);
```

Comment:
Why exactly this context propagation is not needed anymore?

Reply:
Previously connection failure was detected when `org.jboss.netty.channel.ChannelFutureListener#operationComplete` was called with a throwable (yes this really is rather arbitrary). As this method was called from a netty thread there was a need to remember the initial context used when establishing connection. Now connection span is started in `Bootstrap. doResolveAndConnect` and ended in the same method or in `ConnectionCompleteListener`, context is passed directly to that listener.